code/datums/elements/embed.dm

```diff
@@ -71,8 +71,16 @@
 	if(!istype(victim) || HAS_TRAIT(victim, TRAIT_PIERCEIMMUNE))
 		return
 
+	var/flying_speed = throwingdatum?.speed || weapon.throw_speed
+
+	if(!forced && (flying_speed < EMBED_THROWSPEED_THRESHOLD && !ignore_throwspeed_threshold)) // check if it's a forced embed, and if not, if it's going fast enough to proc embedding
+		return
```

Comment:
```suggestion
	var/flying_speed = throwingdatum ? throwingdatum.speed : weapon.throw_speed

	if(!forced && (flying_speed < EMBED_THROWSPEED_THRESHOLD && !ignore_throwspeed_threshold)) // check if it's a forced embed, and if not, if it's going fast enough to proc embedding
		return
```

Reply:
I believe the `or` makes flying_speed a bool variable, which probably isn't intended otherwise this will always return the same thing.

but the code below (line 81-82) doesn't use this variable, so I'm not completely sure what's going on

Reply:
same with above